build/ci-build.yml

```diff
@@ -0,0 +1,117 @@
+      - build/ci-build.yml
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.206'
+  - name: 'Project'
+    value: 'Arcus.Template'
+  - name: 'Arcus_Api_BaseUrl'
+    value: 'http://localhost:5000/api/v1'
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration $(Build.Configuration)'
+          - task: CopyFiles@2
+            displayName: 'Copy build artifacts'
+            inputs:
+              contents: '**/?(bin|obj)/**'
+              targetFolder: '$(Pipeline.Workspace)/build'
+          - task: PublishPipelineArtifact@0
+            displayName: 'Publish build artifacts'
+            inputs:
+              targetPath: '$(Pipeline.Workspace)/build'
+              artifactName: Build
+
+  - stage: IntegrationTests
+    displayName: Integration Tests
+    dependsOn: Build
+    condition: succeeded()
+    jobs:
+      - job: RunIntegrationTests
+        displayName: 'Run integration tests'
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DownloadPipelineArtifact@2
+            displayName: 'Download build artifacts'
+            inputs:
+              artifact: 'Build'
+              path: '$(Build.SourcesDirectory)'
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version)) in web API template'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Install template'
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: '-i "src/$(Project).WebApi/bin/Release/*.nupkg"'
+          - task: DotNetCoreCLI@2
+            displayName: 'Create new project from template'
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: 'arcus-webapi -n MyProject -o /MyProject'
+          - task: Docker@1
+            displayName: 'Build Docker image from newly created web API project from template'
+            inputs:
+              dockerFile: /MyProject/Dockerfile
+              imageName: 'myproject:$(Build.BuildId)'
+          - task: Docker@1
+            displayName: 'Run new web API project Docker image from template'
+            inputs:
+              command: 'Run an image'
+              imageName: 'myproject:$(Build.BuildId)'
+              containerName: 'myproject'
+              ports: '5000:5000'
+              envVars: |
+                ARCUS_HTTP_PORT=5000
```

Comment:
We are explicitly specifying `5000` 4 times, how about a variable?

Reply:
:+1: